services/catalog/src/simcore_service_catalog/core/settings.py

```diff
@@ -47,20 +48,20 @@ class ApplicationSettings(BaseApplicationSettings, MixinLoggingSettings):
     LOG_LEVEL: LogLevel = Field(
         LogLevel.INFO.value,
         validation_alias=AliasChoices(
-            "CATALOG_LOG_LEVEL", "CATALOG_LOGLEVEL", "LOG_LEVEL", "LOGLEVEL"
+            "CATALOG_LOGLEVEL", "LOG_LEVEL", "CATALOG_LOG_LEVEL", "LOGLEVEL"
         ),
     )
     CATALOG_LOG_FORMAT_LOCAL_DEV_ENABLED: bool = Field(
         default=False,
         validation_alias=AliasChoices(
-            "CATALOG_LOG_FORMAT_LOCAL_DEV_ENABLED", "LOG_FORMAT_LOCAL_DEV_ENABLED"
+            "LOG_FORMAT_LOCAL_DEV_ENABLED", "CATALOG_LOG_FORMAT_LOCAL_DEV_ENABLED"
         ),
         description="Enables local development log format. WARNING: make sure it is disabled if you want to have structured logs!",
     )
     CATALOG_LOG_FILTER_MAPPING: dict[LoggerName, list[MessageSubstring]] = Field(
         default_factory=dict,
         validation_alias=AliasChoices(
-            "CATALOG_LOG_FILTER_MAPPING", "LOG_FILTER_MAPPING"
+            "LOG_FILTER_MAPPING", "CATALOG_LOG_FILTER_MAPPING"
```

Comment:
If the order matters. ... if both are defined, which one has precedence here? It should resolve in `CATALOG_LOG_FILTER_MAPPING` provided that both are valid!
Are there tests for these? if not, it would be good to have tests for those having aliases

Reply:
There are tests. The test_cli_list will show you the answer.